api.planx.uk/tests/mocks/applicationTypeCheckMocks.ts

```diff
@@ -0,0 +1,106 @@
+const setValueApplicationTypeBase = {
```

Comment:
nit: these mocks could be typed as `FlowGraph` to ensure they're the expected data structure.

I currently find them very strange to read because you _haven't_ mocked them as realistic flow data strucutres with a `_root` node, despite your function expecting a `FlowGraph` argument